compiler/luci/pass/src/FuseBCQPass.cpp

```diff
@@ -25,6 +25,85 @@
 namespace
 {
 
+bool is_fusable_const(luci::CircleConst *before, luci::CircleConst *after, bool do_w_x)
+{
+  if (after->dtype() != loco::DataType::FLOAT32)
+    return false;
+
+  if (after->rank() != 2)
+    return false;
+
+  if (after->size<loco::DataType::FLOAT32>() != before->size<loco::DataType::FLOAT32>())
```

Comment:
`before` should be checked if its `dtype` is `FLOAT32` as `after` does.

Reply:
This check is needed but not here, I will add this validation :)